src/tooltip.js

```diff
@@ -100,6 +100,7 @@ class Tooltip extends Component<Props, State> {
     super(props);
 
     this.state = {
+      isAnimating: props.isVisible,
```

Comment:
```suggestion
      initialInteractionsComplete: false,
```

I do not think this piece of state should rely on the `isVisible` prop